clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -1349,19 +1349,19 @@ protected static boolean shouldBuildZstdCompressionDictionary(
       // repush from kafka: Existing dictionary will be collected from kafka if found,
       // but will not be built again. This is already checked before calling this function.
       // This is a defensive check.
-      LOGGER.info("No compression dictionary will be generated as the push type is repush");
+      LOGGER.warn("No compression dictionary will be generated as the push type is repush");
```

Comment:
The logging in this function need not be a `warn` as they are expected scenarios, hence, `info` seems more appropriate to me. (Also for end users, WARN tend to be signs of errors, and it leads to misattribution of actual errors)